onto_tool/onto_tool.py

```diff
@@ -1181,7 +1183,7 @@ def format_value(v):
         # Extend the width of each column to contain the longest value.
         max_length = [max(a, b) for a, b in zip(max_length, [len(s) for s in as_text])]
         rows.append(as_text)
-    row_format = " ".join(f"{{:{length}.{length}}}" for length in max_length) + "\n"
+    row_format = "||" + "| ".join(f"{{:{length}.{length}}}" for length in max_length) + "||\n"
     rows.sort(key=lambda x: x[0])
```

Comment:
@pwin Could this `sort` be the reason your `ORDER BY` didn't work for the table?